### PR TITLE
Fixes relating to Remy's comments on issue #113

### PR DESCRIPTION
**Pull request by 6a6d74**

### CODE REVIEW

**Comment by 6a6d74** [comment]
Once reviewed/approved, please SQUASH and merge these commits.

**Comment by 6a6d74** [comment]
BTW - @tomkralidis ... not quite sure why the "cross references" commit ([here](https://github.com/wmo-im/wis2-guide/pull/121/commits/57255d7efd588c9fd51071177b40b07b236bbcbc)) is repeated here. I really only wanted to include the material changes in "Fixes relating to Remy's comments..." commit ([here](https://github.com/wmo-im/wis2-guide/pull/121/commits/20d61e614fe84de4f5c2e3c1a2d944b48de13ec2)) ... hopefully no issues to resolve :)

File: guide/sections/part1/data-consumer.adoc [hunk 1]
```diff
@@ -29,19 +27,19 @@ Notifications from WIS2 Nodes and Global Caches are published on different topic
...
+Data Consumers need to consider their strategy for managing these duplicate messages. From a data perspective, it does not matter which Global Cache instance is used – they will all provide an identical copy of the data object published by the originating WIS2 Node. The simplest strategy is to accept the first notification message and download from the Global Cache instance that the message refers to (i.e., using a URL for the data object at that Global Cache instance). Alternatively, a Data Consumer may have a preferred Global Cache instance, for example, that is located in their region. Whichever Global Cache instance is chosen, Data Consumers will need to implement logic to discard duplicate notification messages based on ``id`` and ``data_id``.
```
Comment (golfvert) [inline guide/sections/part1/data-consumer.adoc:42]: If I'm being pedantic...

to discard duplicate notification messages based on ``id`` and duplicate data objects based on ``data_id``.

**Review by golfvert — CHANGES_REQUESTED** [review]
On small comment left. Make clear that data_id is not a notification message.

File: guide/sections/part1/data-consumer.adoc (hunk 1 above)
Comment (6a6d74) [reply guide/sections/part1/data-consumer.adoc]: Text updated as per suggestion.

**Comment by 6a6d74** [comment]
@golfvert - fixed; request re-review

**Review by golfvert — APPROVED** [review]
OK

**Pull request merged**